akka-actor/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -21,3 +21,24 @@ ProblemFilters.exclude[MissingClassProblem]("akka.actor.dungeon.UndefinedUidActo
 
 # Protect internals against starvation #23576
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.dispatch.Dispatchers.this")
+
+# Remove internal Akka fork of FJP, was meant-to-be internal
+ProblemFilters.exclude[Problem]("akka.dispatch.forkjoin.*")
+
+# Consequences of removing the internal FJP is that class hierarchies changes. Many of those APIs are internal, others are not
+# but probably not meant to be used standalone so it's probably ok. The changes would probably only be observable if you
+# also referenced classes from akka.dispatch.forkjoin
```

Comment:
yes, I can't see much risk with those. 

Maybe we could look closer at what happens if a third-party library is `MonitorableThreadFactory.newThread` via `ExtendedActorSystem.threadFactory.newThread`. If that library was built with Akka 2.5.22 can it still run with Akka 2.6 (this change)?

If that is a problem we could just mention it in migration guide.

Reply:
I tried it out but couldn't find ways to crash it. `threadFactory` returns a `ThreadFactory` and `newThread` a `Thread`, so it's hard to see how internal-ish types could escape there without casting these to internal things?

Reply:
excellent, thanks for trying